warp-core/src/test/scala/com/workday/warp/persistence/mysql/WarpSlickDslSpec.scala

```diff
@@ -275,7 +275,7 @@ class WarpSlickDslSpec extends WarpJUnitSpec with CorePersistenceAware with Skip
   def returnUNIXTimeStamp(): Unit = skipIfH2 {
     val testExecution: TestExecutionRowLike = this.persistenceUtils.createTestExecution(methodSignature1, Instant.now(), 1.0, 10)
     val timeStamp: Rep[Timestamp] = testExecution.startTime
-    val query: Rep[Long] = timeStamp unixTimestamp()
+    val query: Rep[Long] = (timeStamp).unixTimestamp()
```

Comment:
were these changes just a style thing for this class?

Reply:
no, I actually ran into a compile error with scala 2.13:

`[Error] /Users/ruiqi.wang/warp-code/warp-core/warp-core/src/test/scala/com/workday/warp/persistence/mysql/WarpSlickDslSpec.scala:278: can't supply unit value with infix notation because nullary method unixTimestamp: (): slick.lifted.Rep[Long] takes no arguments; use dotted invocation instead: com.workday.warp.persistence.mysql.WarpMySQLProfile.api.TimeStampExtensions(timeStamp).unixTimestamp()`

Reply:
oh shoot that's good to know